README.md

```diff
@@ -97,9 +97,7 @@ Optional features:
 
 ## Rust Version Compatibility Policy
 
-DataFusion's Minimum Required Stable Rust Version (MSRV) policy is to support
-each stable Rust version for 6 months after it is
-[released](https://github.com/rust-lang/rust/blob/master/RELEASES.md). This
-generally translates to support for the most recent 3 to 4 stable Rust versions.
+DataFusion's Minimum Required Stable Rust Version (MSRV) policy is to support stable [4 latest
+Rust versions](https://releases.rs) OR versions within last 4 months, whichever is longer.
```

Comment:
Do [1.80.0](https://releases.rs/docs/1.80.0/) [1.80.1](https://releases.rs/docs/1.80.1/)  count as one release (1.80.x)  or two?

Reply:
As written I think they count for 1

How about this for a slightly different wording

> DataFusion's Minimum Required Stable Rust Version (MSRV) policy is to support the [last 4 minor Rust  versions](https://releases.rs) OR the stable minor Rust version as of 4 months , whichever is lower.
>
> For example, given the releases `1.78.0`, `1.79.0`, `1.80.0`, `1.80.1` and `1.81.0` DataFusion will support `1.78.0`, which is 3 minor versions prior to the most minor recent, `1.81`.

Reply:
Since we specify only major version, it should pick the latest hotfix automatically, right? we just need to mention it on the separate note